setup.py

```diff
@@ -3,15 +3,15 @@
 
 setup(
     name = 'doaj',
-    version = '1.3.0',
+    version = '2.11.11',
     packages = find_packages(),
     install_requires = [
-        "werkzeug==0.8.3",
-        "Flask==0.9",
-        "Flask-Login==0.1.3",
-        "Flask-WTF==0.8.3",
-        "Flask-Mail==0.9.0",
-        "requests==1.1.0",
+        "werkzeug==0.14.1",
+        "Flask==1.0.1",
+        "Flask-Login==0.4.1",
+        "Flask-WTF==0.14.2",
```

Comment:
Oh dear, this line means testing the forms somehow

Reply:
it does. Since I've also changed how the form components and validators are imported, it's critical we test.